### DISCUSSION TITLE
nightscout on google cloud installation node.js script

### DISCUSSION THREAD

**Discussion by nirwer** [NightscoutFoundation/xDrip#3271] [General]
on google cloud nightscout installation, the following message was received:

SCRIPT DEPRECATION WARNING                    

  
  This script, located at https://deb.nodesource.com/setup_X, used to
  install Node.js is deprecated now and will eventually be made inactive.

  Please visit the NodeSource distributions Github and follow the
  instructions to migrate your repo.
  https://github.com/nodesource/distributions

  The NodeSource Node.js Linux distributions GitHub repository contains
  information about which versions of Node.js and which Linux distributions
  are supported and how to install it.
  https://github.com/nodesource/distributions

                          SCRIPT DEPRECATION WARNING

**Comment by Navid200** [comment]
Yes, we are still using Node 14.
Node 16 is extremely slow.  When Nightscout becomes compatible with Node 18, we will switch to it.

Thanks for the reminder.  I will add this note to the guide so that others who may have the same question can find the answer in the guide as well.
